### PR TITLE
Issue #1302: Remove ansible_ssh_user variable to avoid upstream bugs

### PR DESCRIPTION
**Pull request by oxyc**
Let's see what travis says.

### CODE REVIEW

**Comment by geerlingguy** [comment]
@oxyc - See also, the latest comments on https://github.com/ansible/ansible/issues/23530#issuecomment-295878317 - Now _I'm_ getting confused :P

File: provisioning/vars/main.yml [hunk 1]
```diff
@@ -5,4 +5,4 @@ _devtool_docroots:
   - "{{ php_xhprof_html_dir }}"
   - "{{ dashboard_install_dir|default('') }}"
 
-drupalvm_user: "{{ ansible_ssh_user | default(ansible_env.SUDO_USER, true) | default(ansible_env.USER, true) | default(ansible_user_id) }}"
+drupalvm_user: "{{ ansible_env.SUDO_USER | default(ansible_env.USER, true) | default(ansible_user_id, true) }}"
```
Comment (oxyc) [inline provisioning/vars/main.yml]: Why does git delete that one space after `default(ansible_env.SUDO_USER, true)` when it doesn't need to? I almost thought it was some non-space space but hexdump seems correct.

**Comment by geerlingguy** [comment]
Welp, Travis CI is happy... seems safe enough to merge, what do you think?

**Comment by oxyc** [comment]
I think so to but let me verify a few builds locally.

**Comment by geerlingguy** [comment]
K, I also have my drupalvm-live environment I'm going to test it out on in a few minutes (testing an unrelated change right now).

**Comment by oxyc** [comment]
Verified it works with both `ansible_local` and `ansible` provisioners on Ubuntu 12.04 so far. Pretty sure it works, I dont see why the environment variable wouldnt be available...

**Comment by geerlingguy** [comment]
Well that's no fun... my little Drupal role 1.9.0 update is causing:

```
TASK [geerlingguy.drupal : Install Drupal with drush.] *************************
ERROR! Unexpected Exception: '0242ac11-0002-bc99-c30a-000000002437'
to see the full traceback, use -vvv
```

**Comment by geerlingguy** [comment]
See: https://github.com/geerlingguy/drupal-vm/issues/1313

**Pull request merged**